setup/sql/5_3_hotfixes.php

```diff
@@ -106,4 +106,20 @@
 <#7>
 <?php
 $ilCtrlStructureReader->getStructure();
-?>
\ No newline at end of file
+?>
+<#8>
+<?php
+//removes org units from trash
```

Comment:
@bseglias could you add a link to the mantis-ticket 0020168 here in this comment? Thanks!